vispy/visuals/graphs/layouts/networkx_layout.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.#!/usr/bin/env python3
+from ..util import _straight_line_vertices, issparse
+import numpy as np
+try:
+    import networkx as nx
+except:
+    import warnings
+    warnings.warn("Networkx not found, please install network to use its layouts")
+    nx = None
+
+class NetworkxCoordinates:
+    def __init__(self, graph, layout=None, *args, **kwargs):
+        """
+        Converts :graph: into a layout. Can be used in conjunction with networkx layouts or using raw 2D-numpy arrays.
+
+        Parameters
+        ----------
+        graph : a networkx graph.
+        layout : str or dict or iterable-object of float32, optional
+        - When :layout: is s string, a lookup will be performed in the networkx avaiable layouts.
+        - When :layout: is a dict, it will be assumed that it takes the shape (key, value) = (node_id, 2D-coordinate).
+        - When :layout: is numpy array it is assumed it takes the shape (number_of_nodes, 2).
+
+
+        Yields
```

Comment:
I've never seen `Yields` as part of a numpy docstring. I could understand it for a generator, but is this expected for an `__init__` method? Shouldn't this be `Returns` on the `__call__` method?

Reply:
done